snippets/scripts/components-forge-field-to-compass-components/migrateCompassCFToComponent.py

```diff
@@ -329,6 +350,12 @@ def main():
     check_input(API_TOKEN, "apiToken")
     PROJECT_KEY = input("Enter your project key : ").strip()
     check_input(PROJECT_KEY, "projectKey")
+    PREFERENCE_FOR_ALL_OR_ONE_PROJECT = input(("Do you want to migrate Compass custom field values all in once? "
```

Comment:
The way it's phrased now is a bit unclear what each option does. I think it might be clearer to ask:
```
Enter an option to select which issues to migrate:
1. Only the issues in PROJECT_KEY
2. All issues
Enter 1 or 2: 
```

Reply:
Great suggestion! I will update it.